config.json

```diff
@@ -34,10 +35,13 @@
       "username": "mender-bottom",
       "secrets": "mender-secrets.json",
       "trusted": ["oleorhagen", "kacf", "pasinskim",  "olehermanse"],
-      "reviewers": ["oleorhagen", "kacf"],
+      "reviewers": ["oleorhagen"],
```

Comment:
**Note:**

1. The pinging for reviewers feature, which this list is used for, is now disabled by default. You have to add `"ping_reviewer"` to `"bot_features"` to enable it.
2. If it was enabled, this is just the fallback / default list. There is a list for the mender client repo below, where you can also remove kristian, if you want.